examples/image-describer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use PhpLlm\LlmChain\Chain;
+use PhpLlm\LlmChain\Message\Message;
+use PhpLlm\LlmChain\Message\MessageBag;
+use PhpLlm\LlmChain\OpenAI\Model\Gpt;
+use PhpLlm\LlmChain\OpenAI\Model\Gpt\Version;
+use PhpLlm\LlmChain\OpenAI\Runtime\OpenAI;
+use Symfony\Component\HttpClient\HttpClient;
+
+require_once dirname(__DIR__).'/vendor/autoload.php';
+
+$runtime = new OpenAI(HttpClient::create(), getenv('OPENAI_API_KEY'));
+$llm = new Gpt($runtime, Version::gpt4oMini());
+
+$chain = new Chain($llm);
+$messages = new MessageBag(
+    Message::forSystem('You are an image analyzer that looks to images like a comedian would like.'),
+    Message::ofUser(
+        'Describe the image as a comedian would do it.',
+        'https://upload.wikimedia.org/wikipedia/commons/thumb/3/31/Webysther_20160423_-_Elephpant.svg/350px-Webysther_20160423_-_Elephpant.svg.png',
+        'https://upload.wikimedia.org/wikipedia/commons/thumb/3/37/African_Bush_Elephant.jpg/320px-African_Bush_Elephant.jpg',
```

Comment:
based on the code in this PR; this example looks wrong to me

Reply:
Why? Works fine and for me it looks good. Maybe grammar errors.

Reply:
Wouldn't we need to use `Image()` here or what am I missing?

Reply:
Yeah! Thanks for the hint. Missed out to change the example after the last interface change.